views/Pokedex/Boomsweeper/ActionSelectionButton.jsx

```diff
@@ -0,0 +1,71 @@
+export const ACTION = {
+  mark_as_safe: "mark_as_safe",
+  mark_as_seen: "mark_as_seen",
+  mark_as_caught: "mark_as_caught",
+  flag: "flag",
+  reveal: "reveal",
+  explode: "explode",
+  reset: "reset",
+};
+
+export default function ActionSelectionButton({
+  action,
+  currentAction,
+  clickAction,
+}) {
+  const isSelected = action === currentAction;
+
+  const title = (() => {
+    switch (action) {
+      case ACTION.mark_as_safe: return "Mark As Safe";
+      case ACTION.mark_as_seen: return "Mark As Seen";
+      case ACTION.mark_as_caught: return "Mark As Caught";
+      case ACTION.flag: return "Flag";
+      case ACTION.reveal: return "Reveal";
+      case ACTION.explode: return "Explode";
+      case ACTION.reset: return "Reset";
```

Comment:
This is another one where you could just set the values of ACTION to these values. I don't think you're using them in a way where the spaces and capitalization matter so you can save a helper here.

Reply:
Putting spaces in "identifiers" is a strange concept to me, but I suppose here it doesn't really matter.